api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -64,12 +69,32 @@ public Profile getProfile(User user) throws ApiException {
       profile.setDemographicSurveyCompletionTime(user.getDemographicSurveyCompletionTime()
           .getTime());
     }
+    // TODO: Add call to mailchimp using user email hash from DB to get email verified status.
     if (user.getDataAccessLevel() != null) {
       profile.setDataAccessLevel(user.getDataAccessLevel());
     }
     if (user.getAuthorities() != null) {
       profile.setAuthorities(new ArrayList(user.getAuthorities()));
     }
+    String userEmailVerificationStatus = null;
+    if(user.getContactEmail() != null) {
+      try {
+        userEmailVerificationStatus = mailChimpService.getMember(user.getContactEmail());
+      } catch (NotFoundException e) {
+        profile.setEmailVerificationStatus(EmailVerificationStatus.UNVERIFIED);
+      } catch (Exception e) {
+        throw new RuntimeException(e);
+      }
+      if (userEmailVerificationStatus != null) {
+        if (userEmailVerificationStatus.equals("pending")) {
```

Comment:
We should define constants for the statuses we expect.

Reply:
Do we want to define that as a separate file or in our swagger? My instinct is separate file because it is api only. Or we can make them static constants in mailchimpserviceimpl

Reply:
Constants in some Java file is fine.